subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BomSupportPluginsSmokeTest.groovy

```diff
@@ -64,64 +67,66 @@ class BomSupportPluginsSmokeTest extends AbstractSmokeTest {
             module("org.hamcrest:hamcrest-core:1.3").byReason(reason3)
         }
         def springCoreDeps = {
-            module("commons-logging:commons-logging:1.2")
+            module("org.springframework:spring-jcl:5.0.5.RELEASE")
```

Comment:
Spring now uses jcl module (Spring Commons Logging Bridge) instead of plain `commons-logging` since `5.0.0.RELEASE`.